plugins/sentinelone/plugin.spec.yaml

```diff
@@ -2598,6 +2599,77 @@ actions:
         description: SentinelOne API call response data
         type: get_events_response
         required: false
+  update_analyst_verdict:
+    title: Update Analyst Verdict
+    description: Update analyst verdict
+    input:
+      incident_ids:
+        title: Incident IDs
+        description: A list of alert or threat IDs to update the analyst verdict on
```

Comment:
Rewrite
`A list of alert or threat IDs on which we may update the analyst verdict`